File: Lab-6/fibonacci.cpp
#include <iostream>

using namespace std;

int fibonacci(int n){
    if ( n <= 1 ) return n;
    else return fibonacci(n-1) + fibonacci(n-2);
}

int main(){

    string n,m;
    while(cin >> n >> m){
        int f = fibonacci(stoi(n));
        cout << f << endl;
    }
    return 0;
}

File: Lab-6/fibonacci.py
from easyinput import read

def fibonacci(n,m):
    if n<= 1: return n
    else: return fibonacci(n-1,m) + fibonacci(n-2,m)

n,m = read(int),read(int)

while n is not None and m is not None:
    print(fibonacci(n,m))
    n,m = read(int),read(int)

